pd/src/state.rs

```diff
@@ -41,6 +41,14 @@ impl State {
         tracing::info!("running migrations");
         sqlx::migrate!("./migrations").run(&pool).await?;
         tracing::info!("finished initializing state");
+
+        let start = std::time::Instant::now();
+        let mut conn = pool.acquire().await?;
+        query!("SELECT true").fetch_all(&mut conn).await?;
+        let end = start.elapsed();
+        let ms = (end.as_micros() as f64) / 1000.;
+        tracing::info!("no-op query took {}ms ({:.0} per second)", ms, 1000. / ms);
```

Comment:
Non-blocking: Instead of inlining the info into the string, should we report these as fields in the tracing message so they could be mechanically consumed?

Reply:
No objection, but the message isn't intended to be mechanically consumed -- I think that metrics would be better for that than tracing messages -- it's just meant to give a ballpark value that primes our intuition about the cost model for making queries.